lib/inspec/plugin/v2/loader.rb

```diff
@@ -290,7 +290,12 @@ def read_conf_file_into_registry
         when :user_gem
           status.entry_point = status.name.to_s
           status.version = plugin_entry[:version]
-          status.description = fetch_plugin_specs(status.name.to_s)&.summary
+          # Fetch the summary of the gem from local gemspec file instead of remote call using Gem::SpecFetcher.fetcher.
+          unless plugin_entry[:version].nil? #safe check very rare case.
```

Comment:
Hey can you also remove `fetch_plugin_specs` method then? And I think this method is used within the installer too. Can you verify if the use case is similar in that too? If yes, then maybe that could be fixed too. Because yes this would be failing in air-gapped env.

Reply:
Removed it from the `loader.rb`. Inside `installer.rb` that is getting used for `inspec plugin search ` functionality and it's not at least breaking the code right now in air gap environment it just shows an empty search result. May be we can discuss and create different PR for the search if that needs to be worked in an air gap environment. Currently, it seems search needs to look out in rubygems.org to find the given plugin.

Reply:
Yeah true. Cool makes sense!